REST/PowerShell/Feeds/SyncPackages.ps1

```diff
@@ -1,3 +1,93 @@
+[CmdletBinding()]
+param (
+    [Parameter()]
+    [ValidateSet("FileVersions", "LatestVersion", "AllVersions")]
+    [string] $VersionSelection = "FileVersions",
+
+    [Parameter()]
+    [string] $Path,
+
+    [Parameter(Mandatory)]
+    [string] $SourceUrl,
+
+    [Parameter(Mandatory)]
+    [string] $SourceApiKey,
+
+    [Parameter()]
+    [string] $SourceSpace = "Default",
+
+    [Parameter(Mandatory)]
+    [string] $DestinationUrl,
+
+    [Parameter(Mandatory)]
+    [string] $DestinationApiKey,
+
+    [Parameter()]
+    [string] $DestionationSpace = "Default",
```

Comment:
Was copying this over to docs and noticed this spelling - think should this be `$DestinationSpace` (there are 2 occurrences)

Reply:
Thanks Mark.  Corrected.